### DISCUSSION TITLE
why double slashes? https://my.domain//code

### DISCUSSION THREAD

**Discussion by datner** [shlinkio/shlink#2150] [Help wanted] [answered]
### Shlink version

4.1.2

### PHP version

n/a

### How do you serve Shlink

Other (explain in summary)

### Database engine

MySQL

### Database version

n/a

### Summary

When I create a short url-- and regardless of how I create it or how my instance is set up (with trailing slash, without trailing slash, manual code, automatic, all variations I could think of) the result is always including an odd double-slash. I was sure this is a visual bug or something, because even though it reports `https://my.domain//code`, visiting `/code` works just fine. But I could not find any issues on this matter. Am I doing something wrong? Both the UI and the rest api directly show that peculiarity. Statistics show the right url but thats because I publicized the version with the extra slash removed.

What is this quirk? It's soooo annoying visually

**Comment by acelaya** [comment, marked as answer]
The cause is very likely that you added a slash at the end of `DEFAULT_DOMAIN`, or the corresponding config option

**Reply by datner** [reply]
That was my first instinct, but I didn't find anything.. I'll try to check again maybe I glazed over while searching.
Is there a reason why trailing slashes on the base url is not normalized?
